pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,388 @@
+*   [amp-instagram](../../../documentation/components/reference/amp-instagram.md)
+*   [amp-lightbox](../../../documentation/components/reference/amp-lightbox.md)
+*   [amp-selector](../../../documentation/components/reference/amp-selector.md)
+*   [amp-social-share](../../../documentation/components/reference/amp-social-share.md)
+*   [amp-timeago](../../../documentation/components/reference/amp-timeago.md)
+*   Video components
+    *   [amp-video](../../../documentation/components/reference/amp-video.md)
+    *   [amp-youtube](../../../documentation/components/reference/amp-youtube.md)
+
+# Bento in action
+
+The example below demonstrates how to include `amp-base-carousel` and `amp-fit-text` in a non-AMP HTML page.
+
+```
+<!DOCTYPE html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8" />
+    <title>AMP Public Radio</title>
+    <link rel="canonical" href="self.html" />
+    <meta
+      name="viewport"
+      content="width=device-width,minimum-scale=1,initial-scale=1"
+    />
+    <style amp-boilerplate>
+      body {
+        -webkit-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+        -moz-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+        -ms-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+        animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+      }
+      @-webkit-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-moz-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-ms-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-o-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+    </style>
+    <noscript
+      ><style amp-boilerplate>
+        body {
+          -webkit-animation: none;
+          -moz-animation: none;
+          -ms-animation: none;
+          animation: none;
+        }
+      </style></noscript
+    >
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script
+      async
+      custom-element="amp-base-carousel"
+      src="https://cdn.ampproject.org/v0/amp-base-carousel-1.0.js"
+    ></script>
+    <script
+      async
+      custom-element="amp-fit-text"
+      src="https://cdn.ampproject.org/v0/amp-fit-text-1.0.js"
+    ></script>
+    <script>
+      (self.AMP = self.AMP || []).push(function(AMP) {
+        AMP.toggleExperiment("amp-base-carousel-bento", true);
+        AMP.toggleExperiment("amp-fit-text-bento", true);
```

Comment:
```suggestion
        AMP.toggleExperiment("bento", true);
```

Reply:
Otherwise:
```suggestion
        AMP.toggleExperiment("bento-carousel", true);
        AMP.toggleExperiment("bento-fit-text", true);
```

Reply:
Sorry, we updated these experiment names since I made this sample 😇

Reply:
No worries, thanks for catching!

Reply:
I think we should just tend to advise to use a single "bento" experiment in all guides. There's another reason for it: some of the runtime is geared to this experiment a bit better.